src/app/info/WikiItems.tsx

```diff
@@ -49,20 +56,27 @@ const WikiItems: React.FC<AccordianProps> = (props) => {
         <>
             {sortedRows.map((row: DbWikiRow) => {
                 return (
-                    <TableSurface key={row.order}>
-                        <Accordion elevation={0}>
-                            <AccordionSummary
-                                expandIcon={<ExpandMoreIcon />}
-                                aria-controls="panel1-content"
-                                id="panel1-header"
-                            >
-                                <h2>{row.title}</h2>
-                            </AccordionSummary>
-                            <AccordionDetails>
-                                {convertContentToElements(row.content)}
-                            </AccordionDetails>
-                        </Accordion>
-                    </TableSurface>
+                    <AccordionWrapper key={row.order}>
+                        <AdminManagerDependent>
+                            <WikiUpdateButton onClick={buttonAlert}>
+                                <UpdateIcon />
+                            </WikiUpdateButton>
+                        </AdminManagerDependent>
+                        <AccordionSurface>
+                            <Accordion elevation={0}>
+                                <AccordionSummary
+                                    expandIcon={<ExpandMoreIcon />}
+                                    aria-controls="panel1-content"
+                                    id="panel1-header"
+                                >
+                                    <h2>{row.title}</h2>
+                                </AccordionSummary>
+                                <AccordionDetails>
+                                    {convertContentToElements(row.content)}
+                                </AccordionDetails>
+                            </Accordion>
+                        </AccordionSurface>
+                    </AccordionWrapper>
```

Comment:
Can we separate this out into its own component? It can stay in the same file, but as it grows I think we can benefit from keeping it separate